README.md

```diff
@@ -11,23 +11,22 @@
 
 ---
 
-HTNN is...
+HTNN (Hyper Trust-Native Network) is Ant Group's internally developed cloud-native cross-tier networking solution, based on Envoy and Istio, and support extending via Go runtime. HTNN architecturally embraces cloud-native standards, supports multi-cluster management and flexible scalability. As a result, it has an ecosystem that supports high development efficiency. By open sourcing HTNN, Ant Group hopes that the community can share its product capabilities and work together to build advanced network products.
```

Comment:
https://en.wikipedia.org/wiki/Network_layer
maybe we should use layer.

Reply:
@taoyuanyuan 
Updated